README.rst

```diff
@@ -170,6 +170,157 @@ EdgePopulation
+
+   # get all spikes betwen tstart and tstop
+   >>> population.get(0.2, 1.0)
+   [(2, 0.2), (3, 0.3), (2, 0.7)]
+
+   # get spikes attribute sorting (by_time, by_id, none)
+   >>> population.sorting
+   'by_time'
+
+
+SomaReportReader
+++++++++++++++++
+
+.. code-block:: pycon
+
+   >>> somas = libsonata.SomaReportReader('path/to/H5/file')
+
+   # list populations
+   >>> somas.get_populations_names()
+
+   # open population
+   >>> population_somas = somas['<name>']
+
+
+SomaReportPopulation
+++++++++++++++++++++
+
+.. code-block:: pycon
+
+   # get times (tstart, tstop, dt)
+   >>> population_somas.times
+   (0.0, 1.0, 0.1)
+
+   # get unit attributes
+   >>> population_somas.time_units
+   'ms'
+   >>> population_somas.data_units
+   'mV'
+
+   # node_ids sorted?
+   >>> population_somas.sorted
+   True
+
+   # get the DataFrame of the node_id values for the timesteps between tstart and tstop
+   >>> data_frame = population_somas.get(node_ids=[13, 14], tstart=0.8, tstop=1.0)
+
+   # get the data values
+   >>> data_frame.data
+   [[13.8, 14.8], [13.9, 14.9]]
+
+   # get the list of timesteps
+   >>> data_frame.times
+   [0.8, 0.9]
+
+   # get the list of node ids
+   >>> data_frame.ids
+   [13, 14]
+
+
+Pandas can be used to create a dataframe using the data, ids and times lists
+
+.. code-block:: pycon
+
+   >>> import pandas
+
+   df = pandas.DataFrame(data_frame.data, columns=data_frame.ids, index=data_frame.times)
+   print(df)
+          13    14
+   0.8  13.8  14.8
+   0.9  13.9  14.9
+
+
+ElementReportReader
++++++++++++++++++++
+
+.. code-block:: pycon
+
+   >>> elements = libsonata.ElementReportReader('path/to/H5/file')
+
+   # list populations
+   >>> elements.get_populations_names()
+
+   # open population
+   >>> population_elements = elements['<name>']
+
+
+ElementReportPopulation
++++++++++++++++++++++++
+
+.. code-block:: pycon
+
+   # get times (tstart, tstop, dt)
+   >>> population_elements.times
+   (0.0, 4.0, 0.2)
+
+   # get the DataFrame of the node_id values for the timesteps between tstart and tstop
+   >>> data_frame = population_elements.get(node_ids=[13, 14], tstart=0.8, tstop=1.0)
+
+   # get the data values (map of [node_id, element_id] -> values[timesteps]
+   >>> data_frame.data
+   [[46.0, 46.1, 46.2, 46.3, 46.4, 46.5, 46.6, 46.7, 46.8, 46.9], [56.0, 56.1, 56.2, 56.3, 56.4, 56.5, 56.6, 56.7, 56.8, 56.9]]
+
+   # get the list of timesteps
+   >>> data_frame.times
+   [0.8, 1.0]
+
+   # get the list of (node id, element_id)
+   >>> data_frame.ids
+   [(13, 30), (13, 30), (13, 31), (13, 31), (13, 32), (14, 32), (14, 33), (14, 33), (14, 34), (14, 34)]
+
+
+The same way than with soma reports, pandas can be used for better usability
+
+.. code-block:: pycon
+
+   >>> import pandas
+
+   df = pandas.DataFrame(data_frame.data, columns=data_frame.ids, index=data_frame.times)
+   print(df)
+        (13, 30)   (13, 30)   (13, 31)   (13, 31)   (13, 32)  (14, 32)   (14, 33)   (14, 33)   (14, 34)   (14, 34)
```

Comment:
this will change to : 
```
          13                                                                                 14                                            
          30               30               31               31                32    32               33               33               34               34
0.8  46.0  46.099998  46.200001  46.299999  46.400002  46.5  46.599998  46.700001  46.799999  46.900002
1.0  56.0  56.099998  56.200001  56.299999  56.400002  56.5  56.599998  56.700001  56.799999  56.900002

```

Reply:
That s with the MultiIndex

Reply:
rahhh git hub changed the formatting ... 

```
       14                                                13                                            
       32         33         33         34         34    30         30         31         31         32
0.8  46.5  46.599998  46.700001  46.799999  46.900002  46.0  46.099998  46.200001  46.299999  46.400002
1.0  56.5  56.599998  56.700001  56.799999  56.900002  56.0  56.099998  56.200001  56.299999  56.400002

```

Reply:
haha dont worry, i will run the test and copy the output :D